Header header
uint32 height
uint32 width
string image
string serial_number
string camera
string bayer_encoding
string color_encoding
uint32 image_type
uint32 step
uint8[] r
uint8[] g
uint8[] b
uint8[] raw
uint32 border_top
uint32 border_bottem
uint32 border_left
uint32 border_right
float64 focalX
float64 focalY
float64 centerX
float64 centerY
float64 translationX
float64 translationY
float64 translationZ
float64 rotationX
float64 rotationY
float64 rotationZ
